client/elements/suttaplex/sc-suttaplex.html

```diff
@@ -506,7 +506,24 @@ <h3 class="suttaplex-heading">[[_computeMainHeading(item)]]</h3>
           }
 
           _getAcronymOrUid(item) {
-              return item.acronym || item.uid;
+              let sc_acronym = item.acronym;
+              if (item.acronym) {
+                sc_acronym = item.acronym.split('//')[0];
+              }
+              return sc_acronym || item.uid;
+          }
+
+          _getAcronymTitle(item,localize) {
+              let sc_acronym_title = localize('suttaCentralID');
+              if (item.acronym) {
+                const alt_number = item.acronym.split('//')[1];
+                let book = '';
+                if (alt_number) {
+                  (alt_number[0] == 'T') ? book = 'Taishō' : book = 'PTS';
```

Comment:
1. Please use `===` instead of `==`.
2. This has too much logic for a ternary operator IMHO.